### ISSUE TITLE
org-gcal-fetch gives (error . "exited abnormally with code 1")

### ISSUE THREAD

**Issue by jackjackk** [myuhe/org-gcal.el#46]
Getting:
```
Maked:/working/.emacs.d/org-gcal/.org-gcal-token
mapconcat: Wrong type argument: integerp, nil
Got error: (error . "exited abnormally with code 1
")
```
in *Messages* when running org-gcal-fetch.

**Comment by MelleB** [comment]
Same here when running `org-gcal-fetch`

```
Makec:/Users/[USERNAME]/.emacs.d/org-gcal/.org-gcal-token
mapconcat: Wrong type argument: integerp, nil
Got error: (error . "exited abnormally with code 60
")
```

GNU Emacs 24.4.1 (i686-pc-mingw32) of 2014-10-24 on LEG570
Org-mode version 8.3.3
Org-gcal version 20151230.124

**Comment by fzhang00** [comment]
Same here!. 

Make/home/lm172/.emacs.d/org-gcal/.org-gcal-token [2 times]
mapconcat: Wrong type argument: integerp, nil
Got error: (error . "exited abnormally with code 77

**Comment by fzhang00** [comment]
My issue was resovlved by doing the following:

capath=/etc/ssl/certs/
cacert=/etc/ssl/certs/ca-certificates.crt

The author might consider to add curl to the pre-requisite of this software, which I did not know at the beginning.

**Comment by howdoicomputer** [comment]
Yeah, I'm getting `Got error: (error . "exited abnormally with code 1
")` as well.

**Comment by lislon** [comment]
I got same problem on windows using curl from [gow](https://github.com/bmatzelle/gow) (lightweight cygwin). Solved by adding  `_curlrc ` file to %HOME% directory:
```
capath=c:\path\to\ssl\certs
cacert=c:\path\to\ssl\certs\ca-bundle.crt
```

**Comment by bobberb** [comment]
This error is likely due to Windows 10 providing curl.exe in System32 (GLOBAL PATH). Update your Windows PATH to set c:\cygwin64\bin or x:\y\z\curl.exe (cygwin build) in a higher priority, log in and log out. 


You should then receive 77 error, point it to your ca-bundle.crt as @lislon says.

**Comment by bobberb** [comment]
Got error: (error . "exited abnormally with code 1 ") = cURL unsupported protocol
Got error: (error . "exited abnormally with code 2 ") = cURL unable to initialize
Got error: (error . "exited abnormally with code 77 ") = cURL unable to find capath+cacert.

**Comment by telotortium** [comment]
Hi everyone! This repository is obsolete - the actively developed fork is now https://github.com/kidd/org-gcal.el, which is also the "org-gcal.el" on MELPA (see [#115](https://github.com/myuhe/org-gcal.el/issues/115)). I'm one of the maintainers of https://github.com/kidd/org-gcal.el. I can't directly close issues or PRs on this repo, but please download https://github.com/kidd/org-gcal.el and see if your issue or PR is still applicable. If so, open an issue or PR on https://github.com/kidd/org-gcal.el. In any case, you should close your issue or PR in this repo.
